runtime/jcl/common/attach.c

```diff
@@ -204,7 +204,7 @@ Java_openj9_internal_tools_attach_target_IPC_isUsingDefaultUid(JNIEnv *env, jcla
 	/* all offsets are byte offsets */
 	U_32* PSATOLD_ADDR = (U_32 *)(UDATA) 0x21c;  /* z/OS Pointer to current TCB or zero if in SRB mode. Field fixed by architecture. */
 	U_32 tcbBase; /* base of the z/OS Task Control Block */
-	const TCBSENV_OFFSET = 0x154; /* offset of the TCBSENV field in the TCB.  This field contains a pointer to the ACEE. */
+	const int TCBSENV_OFFSET = 0x154; /* offset of the TCBSENV field in the TCB.  This field contains a pointer to the ACEE. */
```

Comment:
TCBSENV_OFFSET is used with expression involving `U_32`, this should be `const U_32`

Reply:
Done, and moved down to order alphabetically with the other `const` instantiations.